dungeon/src/contrib/level/generator/graphBased/levelGraph/LevelNode.java

```diff
@@ -22,10 +22,16 @@
  * neighboring nodes, and the index in the array indicates the {@link Direction} through which the
  * nodes are connected.
  */
-public final class LevelNode {
+public class LevelNode {
 
   static final int MAX_NEIGHBOURS = Direction.values().length;
-  private final Set<Entity> entities;
+
+  /**
+   * A collection of entities stored in this node. Intended to be used to add new entities after
+   * initialization of the {@link LevelNode} class.
```

Comment:
@tgrothe Auch wenn es für `protected` Elemente nicht durch die zukünftige Checkstyle-Regel erforderlich ist: Wenn der Kommentar da ist, sollte er auch den Regeln von Javadoc gehorchen. Also: Erster Satz ist **ein** kurzer Satz, danach ggf. ein Absatz und weitere Erklärungen.

Reply:
Danke. Ja, ich war mir bei `protected` wirklich etwas unsicher, und auch, wie dieser Kommentar geschrieben werden sollte ... Dann ändere ich das noch. :)

Reply:
Ehrlich gesagt würde ich den Kommentar hier **einfach weglassen**:
- Dass es eine Collection von Entities ist, sieht man am Typ.
- Wann die Entities reinkommen, kann man über den Typ nicht ablesen - und das spielt aber auch keine Rolle für den Datentyp und die (alte) Klasse `LevelNode`.

=> Er erklärt hier irgendwie nichts, und passt auch nicht zur alten Klasse `LevelNode` (wo er aber hinzugefügt wurde).


Das gehört als Bemerkung in den Klassenkommentar von `DojoRoom`: Das ist **der zentrale Grund**, warum Du nicht `LevelNode` nutzt, sondern das Ding nochmal verfeinerst.